sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/arithmetic.scala

```diff
@@ -43,6 +43,8 @@ case class UnaryMinus(child: Expression) extends UnaryExpression
 
   override def toString: String = s"-$child"
 
+  override def prettyName: String = "-"
```

Comment:
@HyukjinKwon Should we change "-" to "negative" ?

Reply:
I think it should be `negative` but should show `-` when it's used as the sql operator. You should be able to do this with if-else. e.g.)

```
getTagValue(FunctionRegistry.FUNC_ALIAS).getOrElse("-")
```

You might need to override `sql` IIRC.

Reply:
OK.